src/Yoti.Auth/YotiAttribute.cs

```diff
@@ -1,42 +1,37 @@
 ﻿using System.Collections.Generic;
-using System.Linq;
 using Newtonsoft.Json.Linq;
 using Yoti.Auth.Anchors;
 
 namespace Yoti.Auth
 {
     /// <summary>
-    /// A class to represent a Yoti attribute. A Yoti attribute consists of the attribute name, an associated
+    /// A class to represent a Yoti attribute, extending <see cref="BaseAttribute"/>.
+    /// A Yoti attribute consists of the attribute name, an associated
     /// <see cref="YotiAttributeValue"/>, and a list of <see cref="Anchor"/>s from Yoti.
     /// It may hold one or more anchors, which specify how an attribute has been provided
     /// and how it has been verified within the Yoti platform.
     /// </summary>
-    public class YotiAttribute<T>
+    public class YotiAttribute<T> : BaseAttribute
```

Comment:
I guess in here it's all public to avoid a breaking change...?

Reply:
As this is the return type of the attributes which the clients retrieve, we need to make it public